### ISSUE TITLE
Opacity LUTs support

### ISSUE THREAD

**Issue by NicolasRannou** [FNNDSC/ami#175]
We may want independent opacity LUTs.

Right now the Color LUT contains the opacity information.

**Comment by leht** [comment]
I think the opacity LUTs are already separated from color LUTs. Please see the code in helper.lut.js

**Comment by NicolasRannou** [comment]
yes, the point there that maybe we want a specific `helper.lut.opacity.js` file to live by itself, and provide utilities to "mix" luts (color and opacity)!

Any thought on that? Thanks!
